src/view_element.cpp

```diff
@@ -307,11 +307,9 @@ void View::DrawAccid(DeviceContext *dc, LayerElement *element, Layer *layer, Sta
             int xDiff = x - staff->GetDrawingX();
             rotateOffset = int(xDiff * tan(deg * M_PI / 180.0));
         }
-        
         if (accid->HasFacs() && (m_doc->GetType() == Facs)) {
             y = ToLogicalY(y);
```

Comment:
I understand you need to flip coordinates because the two systems are different. However, it should not happen in the drawing code.

I can merge this but we should still fix the overall issue after. It will be easier to do it separately.